uint16 SelectedSpeed
uint32 SelectedDistance
uint8 Reserved
uint8  SpeedLimitStatus
uint8  SelectedSpeedSource
uint8  Direction
time time_received